src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -50,7 +50,7 @@
     "typescript": "3.5.3"
   },
   "optionalDependencies": {
-    "node-sass": "^4.12.0",
+    "node-sass": "^5.0.0",
```

Comment:
The PR description says:

> This PR updates the version of the caniuse-lite dependency 

However I don't see that change in the diff here. Am I just being confused, or is the description out of date? If the description is out of date, could you update it to clarify? Thanks!

Reply:
I don't mean this to be pedantic BTW. It's so I can meaningfully understand this within the patching approval process.

Reply:
Nope, totally fair question. I realize that the description isn't clear on this.

The `caniuse` dependency is updated in the `package-lock.json` as seen https://github.com/dotnet/aspnetcore/pull/27219/files#diff-4c433fab53ef15d9848bf49770c9f8bebe34733ff8e09f2208966d5a6ad95d75R3148-R3150.

Since the version change is only to the patch version and the caniuse dependency is a transitive dependency, I ran `npm update caniuse-lite` to update the reference in the lock file. The install script will pick up from that when the user runs the poject.

The `node-sass` upgrade is unrelated to the caniuse stuff. This PR fixes two issues in one.

Reply:
Is it legit to make the change only inside `package-lock.json`? What about people using `yarn`?

Reply:
Our templates don't support yarn out of the box, you can install it and deal with it yourself

Reply:
For the main scenario of supporting `dotnet run` working out of the box, just updating the `package-lock.json` should suffice since our build scripts take a dependency on `npm` and not `yarn`.

Reply:
Alright, thanks for clarifying!